src/app/mutation-page/mutation-page.component.ts

```diff
@@ -0,0 +1,54 @@
+import { ChangeDetectionStrategy, Component, inject } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { injectIsMutating, injectMutation } from '@ngneat/query';
+import { HttpClient } from '@angular/common/http';
+import { FormsModule } from '@angular/forms';
+
+interface Todo {
+  id: number;
+  title: string;
+  completed: boolean;
+}
+
+@Component({
+  selector: 'query-mutation-page',
+  standalone: true,
+  imports: [CommonModule, FormsModule],
+  templateUrl: './mutation-page.component.html',
+  styles: [],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class MutationPageComponent {
+  private useIsMutating = injectIsMutating();
+  private useMutation = injectMutation();
+  private httpClient = inject(HttpClient);
```

Comment:
Can we please move the code to the todos service and invoke todosService.addTodo()